cmd/goal/asset.go

```diff
@@ -110,7 +110,7 @@ func init() {
 	addTxnFlags(freezeAssetCmd)
 
 	infoAssetCmd.Flags().Uint64Var(&assetID, "assetid", 0, "ID of the asset to look up")
-	infoAssetCmd.Flags().StringVar(&assetUnitName, "asset", "", "Unit name of the asset to look up")
+	infoAssetCmd.Flags().StringVar(&assetUnitName, "unit", "", "Unit name of the asset to look up")
```

Comment:
I'm not sure about changing the flag name. unit is more accurate, but for I think for most people unit is also a fine name to use.

Reply:
How about following the naming convention stablished in:
```
Usage:
  goal asset create [flags]
...
      --unitname string             Name for the unit of asset
...
```

then, asset/unit would be **unitname**

Reply:
Does it actually look it up by asset (Tether USDt) or unit name (USDt)?

Reply:
It looks up by unit

Reply:
So seems like we should change it or follow Pavel's suggestion, no?